manager-parent/manager-core/src/main/java/com/redhat/service/smartevents/manager/core/api/models/requests/AbstractBridgeRequest.java

```diff
@@ -4,20 +4,31 @@
 
 import javax.validation.constraints.NotEmpty;
 
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+
 import com.fasterxml.jackson.annotation.JsonProperty;
 
 public abstract class AbstractBridgeRequest {
 
     @NotEmpty(message = "Bridge name cannot be null or empty")
     @JsonProperty("name")
+    @Schema(
+            description = "The name of the bridge",
+            example = "bridge1")
     protected String name;
 
     @NotEmpty(message = "Cloud Provider cannot be null or empty.")
+    @Schema(
+            description = "The cloud provider where the bridge resides",
+            example = "aws")
     @JsonProperty("cloud_provider")
     protected String cloudProvider;
 
     @NotEmpty(message = "Region cannot be null or empty.")
     @JsonProperty("region")
+    @Schema(
+            description = "The cloud provider region where the bridge resides",
+            example = "us-east")
```

Comment:
```suggestion
            example = "us-east-2")
```
`us-east` would not work atm. Better to use `us-east-2`